Cannot install make-license version 0.5

Hi, I can not install version 0.5.

```
npm i make-license@^0.4
make-license
```

Works fine.

```
npm i make-license@^0.5
```

Generates error:

```
npm ERR! argv "/home/adabru/.nvm/versions/node/v7.4.0/bin/node" "/home/adabru/.nvm/versions/node/v7.4.0/bin/npm" "i" "make-license@^0.5"
npm ERR! node v7.4.0
npm ERR! npm  v4.0.5
npm ERR! path /home/adabru/Desktop/Apps/word-complete/node_modules/make-license/dist/index.sh
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/home/adabru/Desktop/Apps/word-complete/node_modules/make-license/dist/index.sh'
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/adabru/Desktop/Apps/word-complete/node_modules/make-license/dist/index.sh'
```

I am on Ubuntu 16.04.

--- a/package.json
+++ b/package.json
@@ -3,13 +3,14 @@
   "description": "generate a LICENSE for your open source project",
   "main": "src/index.js",
   "files": [
-    "src"
+    "src",
+    "dist"
   ],
   "scripts": {
     "test": "npm run build && mocha tests/index.test.js -w ",
     "test:single": "npm run build && istanbul cover -x *.test.js,src/index.js _mocha tests/index.test.js",
     "postbuild": "cp src/index.sh dist",
-    "build": "babel src --out-dir dist",
+    "build": "babel src --out-dir dist --presets es2015",
     "build:watch": "babel src --out-dir dist --watch",
     "commit": "git-cz",
     "report-coverage": "cat ./coverage/lcov.info | codecov",

